data/pb/sw-services-calculation/SCBillingSlab.json

```diff
@@ -4,228 +4,178 @@
   "SCBillingSlab": [
     {
       "id": "1",
-      "BuildingType": "RESIDENTIAL",
-      "CalculationAttribute": "No. of water closets",
-      "ConnectionType": "Non Metered",
-      "UOM": "Number",
+      "buildingType": "RESIDENTIAL",
+      "calculationAttribute": "No. of water closets",
+      "connectionType": "Non Metered",
+      "minimumCharge": 0,
       "slabs": [
         {
           "from": 0,
           "to": 1000000000,
-          "charge": 15,
-          "minimumCharge": 0
+          "charge": 15
         }
       ]
     },
     {
       "id": "2",
-      "BuildingType": "RESIDENTIAL",
-      "CalculationAttribute": "No. of toilets",
-      "ConnectionType": "Non Metered",
-      "UOM": "Number",
+      "buildingType": "RESIDENTIAL",
+      "calculationAttribute": "No. of toilets",
+      "connectionType": "Non Metered",
```

Comment:
is connectionType and calculationAttribute code?, if yes then they should be in caps without space

Reply:
No connection type and calculation attribute is not a code. we are not doing any filter on this parameter so.